src/commands/list.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies.
+ */
+const { table } = require( 'table' );
+const chalk = require( 'chalk' );
+const logSymbols = require( 'log-symbols' );
+
+/**
+ * Internal dependencies.
+ */
+const envUtils = require( '../env-utils' );
+const makeCommand = require( '../utils/make-command' );
+const makeDocker = require( '../utils/make-docker' );
+
+// Add command name, alias and description.
+exports.aliases = [ 'ls' ];
+exports.command = 'list';
```

Comment:
Please, add information about this command to the appropriate section of the CHANGELOG.md file.